src/ui/components/Scanner/Scanner.tsx

```diff
@@ -116,7 +157,23 @@ const Scanner = forwardRef(({ setIsValueCaptured }: ScannerProps, ref) => {
             className="qr-code-scanner-icon"
           />
         </IonRow>
+        {OperationType.SCAN_WALLET_CONNECTION === currentOperation && (
+          <IonRow className="actions-button">
+            <IonButton
+              onClick={handlePasteMkId}
+              className="paste-mkid-btn secondary-button"
+              data-testid="paste-meerkat-id-btn"
+            >
+              {i18n.t("scan.pastemeerkatid")}
+            </IonButton>
+          </IonRow>
```

Comment:
Please use `PageFooter` like I showed you in [this PR](https://github.com/cardano-foundation/cf-identity-wallet/pull/451/files). Also look at the `scss` file for customizations.